docs/docsite/rst/community/getting_started.rst

```diff
@@ -21,13 +21,12 @@ Other ways to get involved
 
 Here are some other ways to connect with the Ansible community:
 
-* Find an `Ansible Meetup near me <https://www.meetup.com/topics/ansible/>`_
-   communication
+* Visit the :ref:`Ansible communication guide<communication>`.
+* Find an `Ansible Meetup near me <https://www.meetup.com/topics/ansible/>`_.
```

Comment:
Nitpicking but I wonder if this should be "Find and Ansible meetup near you"?

Reply:
Makes sense to me